lib/Myriad/Transport/Redis.pm

```diff
@@ -32,6 +32,7 @@ has $redis;
 has $redis_pool = [ ];
 has $wait_time = 15_000;
 has $batch_count = 50;
+has $max_pool_count = 10;
```

Comment:
Im not sure if this value is good, or how we should be able to control this value

Reply:
Can you add it to `configure`? I think that would make it configurable by default.